northd/northd.c

```diff
@@ -9438,27 +9438,7 @@ build_dhcpv4_options_flows(struct ovn_port *op,
                 op, lsp_addrs->ipv4_addrs[j].addr,
                 &options_action, &response_action, &ipv4_addr_match)) {
             ds_clear(&match);
-            ds_put_format(
-                &match, "inport == %s && eth.src == %s && "
-                "ip4.src == 0.0.0.0 && ip4.dst == 255.255.255.255 && "
-                "udp.src == 68 && udp.dst == 67",
-                inport->json_key, lsp_addrs->ea_s);
 
-            if (is_external) {
```

Comment:
For logical ports of type "external" we used to reply only if the port was bound locally (`is_chassis_resident()`).  We should not change that behavior.  Moreover, I wonder if we should do that in all cases, also for RENEW/REBIND.

Reply:
doesn't the statement on lines 9455-9458
https://github.com/ovn-org/ovn/blob/aef97ce7739a16c058182647e5f45ac461e26bb4/northd/northd.c#L9455-L9458

still apply the desired behavior?

Reply:
Hmm, isn't that what you're removing here?

Reply:
Oh nevermind, it's fine, both flows had this.  I misread the original code.  Sorry for the noise.